tasks/interactive-bots.md

```diff
@@ -203,21 +203,21 @@ messages to users.
 
 * Check that the bot is working as expected:
 
-  - it reacts to the `@link` and `@resource` at the beginning of a message
+  - it reacts to messages starting with `@link` and `@resource`
 
-  - it sends a private message to the author of the original message
+  - it sends a private message to the sender of the original message
 
 * Take screenshots showing that the bot is working, make sure to have screenshots
 of:
 
  - your terminal window with the bot running, including the command you used to
  run the bot and the output with the bot description
- - messages sent to the bot in the `devel` and `social` streams
+ - messages sent by you to the bot in the `devel` and `social` streams
  - messages sent by the bot in the private messages to the author
  - any other screenshots you find relevant
 
  Add the screenshots to `interactive-bots/links/<username>/`. Make sure your
- filenames include no white spaces or underscores, use dashes (`-`) as connectors.
+ filenames do not have white-spaces. Instead, use dashes (`-`).
```

Comment:
@abhayKashyap03 This is `white space`, not `white-space`, you can check [here](https://www.wikiwand.com/en/Whitespace_character).